crates/sncast/src/main.rs

```diff
@@ -489,6 +492,18 @@ async fn run_async_command(
                 )
                 .await;
 
+                if !import.silent && result.is_ok() && io::stdout().is_terminal() {
+                    if let Some(account_name) =
+                        result.as_ref().ok().and_then(|r| r.account_name.clone())
+                    {
+                        prompt_to_add_account_as_default(account_name.as_str()).unwrap_or_else(
```

Comment:
```suggestion
                        if let Err(err) = prompt_to_add_account_as_default(account_name.as_str()) {
```
you don't use value here so no need to `unwrap`